html2asketch/nodeToSketchLayers.js

```diff
@@ -158,33 +173,43 @@ export default function nodeToSketchLayers(node, options) {
     }
 
     if (boxShadow !== DEFAULT_VALUES.boxShadow) {
-      const shadowObj = shadowStringToObject(boxShadow);
-
-      if (boxShadow.indexOf('inset') !== -1) {
-        if (borderWidth.indexOf(' ') === -1) {
-          shadowObj.spread += parseInt(borderWidth, 10);
+      const shadowObjects = shadowStringToObjects(boxShadow);
+
+      shadowObjects.forEach(shadowObject => {
+        if (boxShadow.indexOf('inset') !== -1) {
+          if (borderWidth.indexOf(' ') === -1) {
+            shadowObject.spread += parseInt(borderWidth, 10);
+          }
+          style.addInnerShadow(shadowObject);
+        } else {
+          style.addShadow(shadowObject);
         }
-        style.addInnerShadow(shadowObj);
-      } else {
-        style.addShadow(shadowObj);
-      }
+      });
     }
 
     // support for one-side borders (using inner shadow because Sketch doesn't support that)
     if (borderWidth.indexOf(' ') === -1) {
       style.addBorder({color: borderColor, thickness: parseInt(borderWidth, 10)});
     } else {
       if (borderTopWidth !== '0px') {
-        style.addInnerShadow(shadowStringToObject(borderTopColor + ' 0px ' + borderTopWidth + ' 0px 0px inset'));
+        style.addInnerShadow(
+          shadowStringToObjects(borderTopColor + ' 0px ' + borderTopWidth + ' 0px 0px inset')[0]
```

Comment:
instead of doing `shadowStringToObjects('…')[0]` lets have two methods `shadowStringToObjects` and `shadowStringToObject`:

- `shadowStringToObjects` will split the shadow string into strings describing single shadows and use `shadowStringToObject` on each of them
- and `shadowStringToObject` will turn each of these strings (representing single shadow) to object

This way, here we will be able to use `shadowStringToObject` and keep things simpler.